data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_15.xml.ep

```diff
@@ -267,5 +267,29 @@ echo -e "[Journal]\\nStorage=persistent" > /etc/systemd/journald.conf.d/01-virt-
       % }
       <script/>
     </post-scripts>
+    <init-scripts t="list">
+      <!-- configure ssh client -->
+      <script>
+        <filename>config_ssh.sh</filename>
+        <source><![CDATA[
+ssh_config_file="/etc/ssh/ssh_config"
+if [ ! -f $ssh_config_file ]; then
+    echo -e "\StrictHostKeyChecking no\nUserKnownHostsFile /dev/null" > $ssh_config_file
+else
+    sed -i 's/#\?\([ \t]\+\)\(StrictHostKeyChecking\)\(.\+\)/\1\2 no/' $ssh_config_file
+    sed -i 's!#\?\([ \t]\+\)\(UserKnownHostsFile\)\(.\+\)!\1\2 /dev/null!' $ssh_config_file
+fi
+mkdir -p -m 700 /root/.ssh
+cat >> /root/.ssh/id_rsa<< EOF
+<%= $get_var->('_SECRET_RSA_PRIV_KEY') %>
+EOF
+sed -i 's/CR/\n/g' /root/.ssh/id_rsa
+chmod 600 /root/.ssh/id_rsa
+echo <%= $get_var->('_SECRET_RSA_PUB_KEY') %> > /root/.ssh/id_rsa.pub
```

Comment:
pub key does not need any character conversion?

Reply:
It need not. It is one line strings in `id_rsa.pub`. Apart from it, `id_rsa` consists of multiple lines texts, but our openqa setting support one line only or multiple line with such length is not user friendly. I refered to @RoyCai7 change. 
And we need note that the replacement charactors "CR" at here I used or Roy's "9A", they may not apply to all keys, if we would change keys in future, either we choose a key which not contains "CR" or "9A", or we replace with another 2 charactors. we don't use a string with more charactors like "CRLN" or else to minimize the risk, because "CRLN" exceeded the max length of the value of an openqa setting. I am going to add a comment.